Action/ConvertPaymentAction.php

```diff
@@ -41,6 +41,7 @@ public function execute($request)
             'email' => $order->getClientEmail(),
             'firstName' => isset($d['firstName']) ? $d['firstName'] : '',
             'lastName' => isset($d['lastName']) ? $d['lastName'] : '',
+            'language' => $order->getLocale() ? substr($order->getLocale(), 0, 2) : '',
```

Comment:
I think idea with interface is good. We can check if storage entity has implemented some "Language interface".  If yes - only than add lang parameter. But I dont think that bundle has to check if such locale is supported or no by PayU. As I wrote, in case if locale is absent or wrong - default will set by PayU.